Assets/Engine/Examine/SingleColorMaterial.mat

```diff
@@ -40,7 +40,7 @@ Material:
         m_Scale: {x: 1, y: 1}
         m_Offset: {x: 0, y: 0}
     - _MainTex:
-        m_Texture: {fileID: 2800000, guid: 1dbe9ceb117c79d47898dfdfc5a960d5, type: 3}
```

Comment:
I don't remember changing any materials. Must've snuck in somehow while I was testing or something.

Reply:
This material seems to get changed on its own fairly often. I think the color changes depending on some variable but if you can avoid committing this file in the future when you see it change itself, that would be preferred.